Makefile

```diff
@@ -110,24 +110,37 @@ all: push
 # push assumes that the building user already has docker credentials
 #  to push to whatever the target repository (specified in $(image)) may be.
 push: image
-	$(DOCKER) push $(image):$(version)
-ifneq ($(ltype),)
-	$(DOCKER) tag $(image):$(version) $(image):$(ltype)
-	$(DOCKER) push $(image):$(ltype)
-endif
-ifneq ($(latest),)
-	$(DOCKER) tag $(image):$(version) $(image):$(latest)
-	$(DOCKER) push $(image):$(latest)
-endif
+	img=$$(echo $(image) | cut -d ',' -f 1) && \
```

Comment:
It would help me if I knew what form `$(image)` had.

Reply:
Comment added to tell you.